README.md

```diff
@@ -306,8 +306,78 @@ However, the main advantage of this approach comes from the fact that we can kee
 
 ### Typechecking
 
-Pretty stnadard, we simply collect all the hashes and the associated types for the modules in scope and then proceed with typechecking
+Pretty standard, we simply collect all the hashes and the associated types for the modules in scope and then proceed with typechecking
 
 ### Evaluation
 
 As discusssed in the pinning section, evaluation is done on a fully typechecked and pinned `Expr`ession.
+
+## Developing Inferno scripts with VScode
+
+In a shell go to `~/inferno` (unless you cloned inferno into a different location) and run `nix build .#vscode-inferno-lsp-server`. Then in VScode press `ctrl+shift+P` and run `Install from VSIX`. In the window, navigate to `~/inferno/result` and select `inferno-lsp-server.vsix`.
+
+Do the same after for the VSIX created using `nix build .#vscode-inferno-syntax-highlighting`.
+
+### Regular Inferno
```

Comment:
Maybe this heading could be rephrased since there's only one set of instructions now